backend-api/src/functions/asyncBiometricToken/responses.ts

```diff
@@ -0,0 +1,13 @@
+import { APIGatewayProxyResult } from "aws-lambda";
+
+export const notImplementedResponse: APIGatewayProxyResult = {
```

Comment:
Thought: I like splitting out responses into their own file. As there's nothing specific to biometricToken about this one, might it be worth putting it in a 'common' space (e.g. in `utils/`, or in its own folder if you can think of something more specific), for ease of re-use?

Reply:
Happier with that! Down with dumping everything in a utils folder.